data/sound-strings/pl.json/localize.json

```diff
@@ -30,6 +30,7 @@
 "in_2_5_kilometers":"Za dwa i pół kilometra",
 "in_3_kilometers":"Za trzy kilometry",
 "then":"Następnie",
+"dist_direction_onto_street":"%1$s %2$s w %3$s",
```

Comment:
```suggestion
"dist_direction_onto_street":"%1$s %2$s na %3$s",
```

Reply:
Other translations that I don't really know how to make into a GitHub review:
"leave_the_roundabout":"Zjedź z ronda.",
"exit":"Zjedź.",

Reply:
This should be a review of sound.txt - sorry

Reply:
> "exit":"Zjedź.",

In which context it is said because I'm not sure if it's wrong or correct here? Provide the example in polish with this.

Reply:
It is used as a verb, not a noun. See https://zyphlar.github.io/organicmaps-locale-viewer/

Some examples that don't really make sense:
```
Jedź prosto. Następnie Zjazd.
Zjazd. Następnie Dojedziesz.
Za trzysta stóp Jedź prosto w Main Street Następnie Zjazd.
```

Reply:
I think it should be improved to say "Zjedź w prawo" because that makes sense in most countries and is probably only used on motorways and trunk roads. Because only this types of roads have no intersections with traffic lights. Then the opposite should be added for "British" countries.

Reply:
FYI this feature is programmed to announce "in 200 meters, turn left onto Street or Exit Name" no matter what kind of street is being turned onto. If the distance is far enough and a name appears at the top of the screen in OrganicMaps, it will be sent to the TTS engine when this feature is enabled by the user (disabled by default.)

I understand that in some countries this behavior could be a little confusing, for example in Japan a lot of navigation is done with named intersections instead of named streets. A future PR will handle such cases as we discover them and consider how to support it.

Also note that this feature will not apply to rapidly executed turns: if there's only a few seconds left before the turn needs to be made, OrganicMaps will say "turn right, then turn left" without a distance or street name.

Reply:
@lunarna-gh can you give me a before/after of the strings from https://zyphlar.github.io/organicmaps-locale-viewer/ which don't make sense and how we can improve each? I don't speak Polish so a clear Incorrect vs Correct example will be best for me to make the necessary code/string changes.

Reply:
There's no need to change sentence ordering or anything, it was just in reply to pm4rcin asking what sounds wrong. These sentences will make sense when translated as stated above

leave_the_roundabout: Zjedź z ronda.
exit: Zjedź.

Reply:
>exit: Zjedź.

Is it used only on highways and trunk roads? Because we could make it the same as Magic Earth. There it says "Wybierz zjazd".

Reply:
I need to test this PR and see how it works when exiting highways. I'll share my findings.

Reply:
> "Wybierz zjazd" kind of sounds like it's a choice up to the driver instead of guiding the driver through the Organic Maps route though, doesn't it?

After some thinking and testing `zjedź` or `zjedź w prawo` is much better. I'd opt for the second because it sounds better in my opinion.

Reply:
I guess `zjedź w prawo` is a bit clearer, but this would require a separate string for countries which drive on the left with a `zjedź w lewo` translation (UK tourists, visitors and polish speakers need to be accounted for) which sounds like a lot of extra effort if `zjedź` works